dhall/src/Dhall/Tutorial.hs

```diff
@@ -2022,823 +1909,95 @@ import Dhall
 -- * Prefer general purpose built-ins or built-ins appropriate for the task of program configuration
 -- * Design built-ins to catch errors as early as possible (i.e. when type-checking the configuration)
 
--- $builtinOverview
---
--- Each of the following sections provides an overview of builtin functions and
--- operators for each type.  For each function you get:
---
--- * An example use of that function
---
--- * A \"type judgement\" explaining when that function or operator is well
---   typed
---
--- For example, for the following judgement:
---
--- > Γ ⊢ x : Bool   Γ ⊢ y : Bool
--- > ───────────────────────────
--- > Γ ⊢ x && y : Bool
---
--- ... you can read that as saying: "if @x@ has type @Bool@ and @y@ has type
--- @Bool@, then @x && y@ has type @Bool@"
---
--- Similarly, for the following judgement:
---
--- > ─────────────────────────────────
--- > Γ ⊢ Natural/even : Natural → Bool
---
--- ... you can read that as saying: "@Natural/even@ always has type
--- @Natural → Bool@"
---
--- * Rules for how that function or operator behaves
---
--- These rules are just equalities that come in handy when reasoning about code.
--- For example, the section on @(&&)@ has the following rules:
---
--- > (x && y) && z = x && (y && z)
--- >
--- > x && True = x
--- >
--- > True && x = x
---
--- These rules are also a contract for how the compiler should behave.  If you
--- ever observe code that does not obey these rules you should file a bug
--- report.
-
--- $bool
---
--- There are two values that have type @Bool@ named @True@ and @False@:
---
--- > ───────────────
--- > Γ ⊢ True : Bool
---
--- > ────────────────
--- > Γ ⊢ False : Bool
---
--- The built-in operations for values of type @Bool@ are:
+-- $prelude
 --
-
--- $or
+-- There is also a Prelude available at:
 --
--- Example:
+-- <https://prelude.dhall-lang.org>
 --
--- > $ dhall
--- > True || False
--- > <Ctrl-D>
--- > Bool
--- > 
--- > True
+-- There is nothing \"official\" or \"standard\" about this Prelude other than
+-- the fact that it is mentioned in this tutorial.  The \"Prelude\" is just a
```

Comment:
This reads like an anti-endorsement to me. IMHO it should be pointed out that the Prelude isn't special in a "technical" way – i.e. no special integration with implementations is necessary to implement it.

But I think we _should_ call it "official" to emphasize that it is curated and should reflect good practices. And since the implementation maintainers vote on Prelude changes, it really is kind of "standard" too, no?

Reply:
I think I'll just remove this paragraph.  If we want to expand on the role of the Prelude we can add a `README` to the Prelude